### PR TITLE
Add doc for InvariantGlobalization property

### PR DESCRIPTION
**Pull request by captainsafia**
Related to https://github.com/dotnet/aspnetcore/issues/47029.

Document the `InvariantGlobalization` property since we intend to set it in templates.

### CODE REVIEW

**Comment by DamianEdwards** [comment]
Looks good! https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#invariantglobalization

File: src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd [hunk 1]
```diff
@@ -6173,5 +6173,11 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  
+  <xs:element name="InvariantGlobalization" type="msb:boolean" substitutionGroup="msb:Property">
+    <xs:annotation>
+      <xs:documentation><!-- _locID_text="InvariantGlobalization" _locComment="" -->Indicates whether an app runs in globalization-invariant mode without access to culture-specific data and behavior.</xs:documentation>
```
Comment (danmoseley) [inline src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:6179]:
```suggestion
      <xs:documentation><!-- _locID_text="InvariantGlobalization" _locComment="" -->Determines whether an app runs in globalization-invariant mode without access to culture-specific data and behavior.</xs:documentation>
```
imo it's clearer

**Review by danmoseley — APPROVED** [review]

File: src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd (hunk 1 above)
Comment (captainsafia) [reply src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:6179]: I did this to follow the pattern in other documented boolean-style properties. They all use "Indicates". The notable excpetion being `Using_Static` which starts with "Determines."

Comment (eerhardt) [inline src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:6179]: (nit) Do we need to specify it is a ".NET Core app" (or whatever the name of that is now days)? This property doesn't work for a .NET Framework application.

https://learn.microsoft.com/en-us/dotnet/core/runtime-config/globalization#invariant-mode

> Determines whether a .NET Core app runs in globalization-invariant mode without access to culture-specific data and behavior.

**Review by eerhardt — APPROVED** [review]

File: src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd (hunk 1 above)
Comment (DamianEdwards) [reply src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:6179]: Do we have such qualifications for other properties defined here that are only applicable in modern .NET apps? We wouldn't say ".NET Core" in documentation now either.

Comment (eerhardt) [reply src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:6179]: All these properties don't work in .NET Framework either:

https://github.com/dotnet/msbuild/blob/1a6d753a7648baf0cc991af0c0d254a41d36eba6/src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd#L2362-L2393

Interestingly, they all refer to "the project" and not an "app".

I think what we have here is fine. Just wanted to check if the qualification was needed or not.

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**